File: Entregables/T5/solve.cpp
#include <iostream>
#include <vector>
#define forn(i,n) for(int i = 0; i < n; i++)

using namespace std;

int solve(vector<int> A){
    int work = 0;
    int contador = 0;
    int N = A.size();
    for(int i=0; i<N; i++){
        contador += A[i];
        work += abs(contador);
    }
    return work;
}

int main(){
    vector<int> res;
    while (true)
    {
        int N;
        cin >> N;

        if (N == 0)
            break;

        vector<int> A(N, 0);
        forn(i,N){
            cin >> A[i];
        }
     
        res.push_back(solve(A));
    }

    forn(i,res.size()){
        cout << res[i] << endl;
    }
    return 0;
}

File: Entregables/T5/solve.py
# Usuario vJudge: sebasAndres

res = []

def solve(A):
    n = len(A)
    cont = 0
    work = 0
    for i in range(n):
        cont += A[i]
        work += abs(cont)    
    return work

while True:
    n = int(input())
    if n==0:
        break

    A = list(map(int, input().split()))
    r = solve(A)
    res.append(r)

for r in res:
    print(r)